chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -299,6 +297,12 @@
 
   private boolean forceReset = false;
 
+  /**
+   * The octree implementation to use
+   */
+  private Octree.ImplementationEnum octreeImplementation
```

Comment:
The enum can just be named `Octree.Implementation`, having the enum end with enum sounds redundant. Or maybe `ImplementationType`?

Reply:
Yeah, the name is not the best, i just didn't want to cause confusion with the interface `Octree.OctreeImplementation`

Reply:
This type does not exist anymore